flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java

```diff
@@ -190,7 +190,7 @@
 
     private final PipelineExecutorServiceLoader executorServiceLoader;
 
-    private final Configuration configuration;
+    protected final Configuration configuration;
```

Comment:
nit: Given that this class has fields `config` and `configuration` (yikes), it would be nice to add a small comment here as well as to what this field holds.

Reply:
I added a comment.